### PR TITLE
remove turbo from storybook

### PR DESCRIPTION
**Pull request by gyfchong**
## Important: Request PR reviews on Slack
Please reach out to the design system team on Slack in `#prod_design_system` for PR reviews. GitHub notifications (e.g. from tagging a person) are not actively monitored.

## Why
Since moving storybook to turbo, we've had issues with chromatic's ability to detect changes.

## What
Revert the changes made in https://github.com/cultureamp/kaizen-design-system/pull/4781

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -14,14 +14,13 @@
...
     "storybook:preview": "NODE_ENV=production pnpm storybook --docs",
     "storybook:clean": "rm -rf storybook/public",
-    "storybook:build": "storybook build -c storybook -o storybook/public",
-    "storybook:build:root": "pnpm turbo storybook:build",
-    "storybook:build:test": "pnpm turbo storybook:build -- --test",
-    "storybook:build:prod": "pnpm turbo storybook:build -- --docs",
-    "test:storybook": "test-storybook --config-dir storybook --skipTags='skip-test' --url http://127.0.0.1:4479",
+    "storybook:build": "storybook build -c storybook -o storybook/public --stats-json",
+    "storybook:build:test": "pnpm storybook:build -- --test",
+    "storybook:build:prod": "pnpm storybook:build -- --docs",
```
Comment (HeartSquared) [inline package.json:22]: Pretty sure the `--` is a turbo thing :)
```suggestion
    "storybook:build:test": "pnpm storybook:build --test",
    "storybook:build:prod": "pnpm storybook:build --docs",
```

File: package.json [hunk 2]
```diff
@@ -14,14 +14,13 @@
     "clean": "pnpm turbo clean && pnpm storybook:clean && rimraf node_modules",
     "test": "pnpm turbo test",
     "test:ci": "pnpm turbo test:ci",
-    "storybook": "storybook dev -p ${STORYBOOK_PORT:-4479} -c storybook",
+    "storybook": "storybook dev -p ${STORYBOOK_PORT:-6006} -c storybook",
```
Comment (HeartSquared) [inline package.json:17]: There's some other places to change I think :)

File: .github/workflows/test-stories.yaml [hunk 3]
```diff
@@ -109,7 +109,7 @@ jobs:
       - uses: ./.github/actions/setup
       - name: Build Storybook for Chromatic
         # We want both stories and docs for the branch preview
-        run: pnpm storybook:build:root -- --webpack-stats-json
+        run: pnpm storybook:build -- --webpack-stats-json
```
Comment (gyfchong) [inline .github/workflows/test-stories.yaml:112]:
```suggestion
        run: pnpm storybook:build --stats-json
```

**Review by HeartSquared — APPROVED** [review]

File: package.json [hunk 4]
```diff
@@ -14,14 +14,13 @@
...
     "storybook:preview": "NODE_ENV=production pnpm storybook --docs",
     "storybook:clean": "rm -rf storybook/public",
-    "storybook:build": "storybook build -c storybook -o storybook/public",
-    "storybook:build:root": "pnpm turbo storybook:build",
-    "storybook:build:test": "pnpm turbo storybook:build -- --test",
-    "storybook:build:prod": "pnpm turbo storybook:build -- --docs",
-    "test:storybook": "test-storybook --config-dir storybook --skipTags='skip-test' --url http://127.0.0.1:4479",
+    "storybook:build": "storybook build -c storybook -o storybook/public --stats-json",
```
Comment (gyfchong) [inline package.json:20]:
```suggestion
    "storybook:build": "storybook build -c storybook -o storybook/public",
```

**Pull request merged**